Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -192,23 +203,23 @@
 				warning('Did not succeed in repeating PLDs for each TE for Hadamard sequence import');
 			end
 			% Make sure that number of volumes can be divided by the repeated PLDs
-			jsonOut.PostLabelingDelay = repeatedPLDs;
+			jsonOut.PostLabelingDelay = repeatedPLDs; %do we want to repeat the PLDs??
 		end
 	end
 end
 
 % Check for FME Hadamard sequences
 if isfield(jsonOut,'SeriesDescription')
-	isHadamardFME = ~isempty(regexp(char(jsonOut.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
-	if isHadamardFME
+	bHadamardFME = ~isempty(regexp(char(jsonOut.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
```

Comment:
Do we need the `char` here? `SeriesDescription` should already be a `char`. Also, should we allow different cases in the regular expression (upper and lower cases) (by `(?i)`)?

Reply:
Different cases aren't necessary. The FME format should be fixed. Series description should always be a char, you're right. Doesn't really hurt though.